.github/workflows/clang.yml

```diff
@@ -43,7 +40,9 @@ jobs:
         ${CXX} --version
         scripts/set-up-python.sh --python=python3
         source .venv/bin/activate
```

Comment:
The `source .venv/bin/activate` line should be redundant now.

Reply:
It is. I removed it, along with the corresponding line in the GCC action.